packages/strapi-connector-bookshelf/lib/buildQuery.js

```diff
@@ -16,7 +16,16 @@ const BOOLEAN_OPERATORS = ['or'];
 const buildQuery = ({ model, filters }) => qb => {
   const joinsTree = buildJoinsAndFilter(qb, model, filters);
 
-  if (_.has(filters, 'where') && Array.isArray(filters.where) && filters.where.length > 0) {
+  const singleFilter = _.has(filters, 'limit') && filters.limit === 1;
```

Comment:
to keep some consistency can you rename to 
`isSingleResult`
`hasJoins`
`hasWhereFilters`
`isDistinctQuery`

Reply:
I think I renamed in the way you wanted